sos/report/plugins/__init__.py

```diff
@@ -2982,6 +2982,9 @@ def add_journal(self, units=None, boot=None, since=None, until=None,
         self._add_cmd_output(cmd=journal_cmd, timeout=timeout,
                              sizelimit=log_size, pred=pred, tags=tags,
                              priority=priority)
+        self.add_cmd_tags({
+            "journalctl --no-pager $": "journal_all",
+        })
```

Comment:
We can just add the new tag directly:

```diff
@@ -63,7 +63,8 @@ class Logs(Plugin, IndependentPlugin):
         journal = any([self.path_exists(self.path_join(p, "log/journal/"))
                        for p in ["/var", "/run"]])
         if journal and self.is_service("systemd-journald"):
-            self.add_journal(since=since, tags='journal_full', priority=100)
+            self.add_journal(since=since, tags=['journal_full', 'journal_all'],
+                             priority=100)
             self.add_journal(boot="this", since=since,
                              tags='journal_since_boot')
             self.add_journal(boot="last", since=since,
```

Reply:
Updated